diff -up ding-libs-0.5.0/Makefile.am.soname ding-libs-0.5.0/Makefile.am
--- ding-libs-0.5.0/Makefile.am.soname	2015-06-23 20:40:31.768363383 +0200
+++ ding-libs-0.5.0/Makefile.am	2015-06-23 20:41:13.996493605 +0200
@@ -142,7 +142,7 @@ libcollection_la_SOURCES = \
     trace/trace.h
 libcollection_la_DEPENDENCIES = collection/libcollection.sym
 libcollection_la_LDFLAGS = \
-    -version-info 5:0:1 \
+    -version-info 3:0:1 \
     -Wl,--version-script=$(top_srcdir)/collection/libcollection.sym
 
 check_PROGRAMS += \
@@ -268,7 +268,7 @@ libini_config_la_LIBADD = \
     libref_array.la \
     libbasicobjects.la
 libini_config_la_LDFLAGS = \
-    -version-info 6:0:1 \
+    -version-info 5:0:2 \
     -Wl,--version-script=$(top_srcdir)/ini/libini_config.sym
 
 dist_noinst_DATA += \
